build/Build.cs

```diff
@@ -184,15 +186,68 @@ public Build()
                                 nugetVersion,
                                 FixedRuntimes.Cloud);
 
-                      DoPublish(RootProjectName, Frameworks.NetCoreApp31, nugetVersion, FixedRuntimes.Portable);
-
                       // Create the self-contained Calamari packages for each runtime ID defined in Calamari.csproj
                       foreach (var rid in Solution?.GetProject(RootProjectName).GetRuntimeIdentifiers()!)
                           DoPublish(RootProjectName, Frameworks.NetCoreApp31, nugetVersion, rid);
                   });
 
+        Target PublishCalamariFlavourProjects => _ => _
+                                                          .DependsOn(Compile)
+                                                          .Executes(() =>
+                                                                    {
+                                                                        var calamariFlavourGroup = MigratedCalamariFlavours.Flavours.Select(flavour => SourceDirectory.GlobFiles($"**/{flavour}*.csproj")); //We need Calamari & Calamari.Tests
+                                                                        foreach (var calamariFlavourProjects in calamariFlavourGroup)
```

Comment:
I wonder if we could make this a little easier to maintain with some Linq instead of nested loops and conditionals?

Reply:
I've updated is a bit let me know what you think